sea-orm-cli/src/commands/migrate.rs

```diff
@@ -104,12 +105,17 @@ pub fn run_migrate_init(migration_dir: &str) -> Result<(), Box<dyn Error>> {
 pub fn run_migrate_generate(
     migration_dir: &str,
     migration_name: &str,
+    universal_time: bool,
 ) -> Result<(), Box<dyn Error>> {
     println!("Generating new migration...");
 
     // build new migration filename
-    let now = Local::now();
-    let migration_name = format!("m{}_{}", now.format("%Y%m%d_%H%M%S"), migration_name);
+    let formatted_now = if universal_time {
+        Utc::now().format("%Y%m%d_%H%M%S")
+    } else {
+        Local::now().format("%Y%m%d_%H%M%S")
+    };
```

Comment:
Sorry for the nitpick :P

```suggestion
    const fmt: &str = "%Y%m%d_%H%M%S";
    let formatted_now = if universal_time {
        Utc::now().format(fmt)
    } else {
        Local::now().format(fmt)
    };
```